### PR TITLE
made call to compute_cartesian_path more pythonic; added links to mov…

### PR DESCRIPTION
**Pull request by jwhendy**
…eit_commander API

I tweaked the tutorial intro, as `MoveGroupCommander` is used just a much as `RobotCommander`, so it seemed fair to highlight both, and I added a link to the API a well, which I found really helpful when getting my hands dirty (just nice to see all the calls one has available).

While I'm new, from what I've seen in python blogs various talks on python, using named arguments seemed more elegant/pythonic than un-named arguments with comments after them, especially since the variables were named perfectly already.

### CODE REVIEW

File: doc/pr2_tutorials/planning/scripts/move_group_python_interface_tutorial.py [hunk 1]
```diff
@@ -216,9 +216,9 @@ def move_group_python_interface_tutorial():
   ## translation.  We will specify the jump threshold as 0.0, effectively
   ## disabling it.
   (plan3, fraction) = group.compute_cartesian_path(
-                               waypoints,   # waypoints to follow
-                               0.01,        # eef_step
-                               0.0)         # jump_threshold
+                               waypoints=waypoints,
+                               eef_step=0.01,
+                               jump_threshold=0.0)
```
Comment (davetcoleman) [inline doc/pr2_tutorials/planning/scripts/move_group_python_interface_tutorial.py]: i love this!

**Comment by davetcoleman** [comment]
i just retriggered the build

**Comment by davetcoleman** [comment]
You have a slight syntax error:

> Warning, treated as error:
/home/travis/build/ros-planning/moveit_tutorials/doc/ros_visualization/visualization_tutorial.rst:100: WARNING: Bullet list ends without a blank line; unexpected unindent.

**Comment by jwhendy** [comment]
I think I've figured this all out...

- #101 was causing build errors based on 404 errors for the links used
- after fixing that, it failed due to using the same hyperlink text for both `pr2_moveit_config` links so I fixed that as well
- I *thought* the issue was due to my inserted image in #91 which added an extra blank line after the bullet list, but then had a real head scratcher when I couldn't find that image anymore! Turns out that #92 removed it
- I also found that #92 replaced the entire bullet list in "Step 3: Interact with the PR2", and added two spaces before an asterisk, causing the error you see above. That said, the comment that was merged [failed](https://travis-ci.org/ros-planning/moveit_tutorials/builds/260290040?utm_source=github_status&utm_medium=notification) due to the non-existent .png file issue you referred to in #94 so it couldn't continue to find this issue...

So! If you take #109, I *think* this will then pass as well.

**Pull request closed without merging**